### PR TITLE
feat: stl move panel

### PR DESCRIPTION
**Pull request by andrushechka-b**
WARNING: This pull request adds the gui move panel containing buttons and editlines dedicated for Translating, Rotating and Scaling. But the functionality has been realized for Translating and Rotating only. Scaling functionality will be added later.

### CODE REVIEW

**Review by l1va — APPROVED** [review]
Looks good to me.
Just small comments/suggestions. 
Also good to have common formatting style - I vote to use standard formatting. You can set it in your editor to format automatically.

**Comment by andrushechka-b** [comment]
I have applied your suggestions and standard formatting style

File: src/figure_editor.py [hunk 1]
```diff
@@ -111,6 +111,62 @@ def __init__(self, on_change: Callable[[Dict[str, float]], None],
...
+                act_pos, act_neg, act_set = methods[col, param]
+                btn_pos.clicked.connect(act_pos)
+                btn_neg.clicked.connect(act_neg)
+                edit.textChanged.connect(act_set)
+                
+            mainLayout.addLayout(gridLayout)
+            
+        initial_pos = [0, 0, 0]
+        initial_orient = [0, 0, 0]
+        initial_sacle = [1, 1, 1]
```
[the remark this answers is not in the document]
Comment (andrushechka-b) [reply src/figure_editor.py]: fixed that

File: src/window.py [hunk 2]
```diff
@@ -484,6 +527,8 @@ def TransformActor(obj, event):
             else:
                 self.boxWidget.SetEnabled(True)
             # self.interactor.GetInteractorStyle().SetCurrentStyleToTrackballActor()
+            
+            self.stl_move_panel.setEnabled(True)
```
[the remark this answers is not in the document]
Comment (andrushechka-b) [reply src/window.py]: I agree, so, I have modified the code as you suggest

**Pull request merged**